keras_cv/models/object_detection/retina_net/retina_net.py

```diff
@@ -297,7 +304,17 @@ def compile(
                 A preconfigured `FocalLoss` is provided when the string "focal" is
                 passed.
             weight_decay: a float for variable weight decay.
-            metrics: object detection metrics.
+            metrics: KerasCV object detection metrics that accept decoded
+                bounding boxes as their inputs.  Examples of this metric type are
+                `keras_cv.metrics.BoxRecall()` and
+                `keras_cv.metrics.BoxMeanAveragePrecision()`.  When `metrics` are
+                included in the call to `compile()`, the RetinaNet will perform
+                non max suppression decoding using a
+                `keras_cv.layers.MultiClassNonMaxSuppression()` layer in the
```

Comment:
Is this guaranteed to be this specific nms layer? What if the user specified a custom one?

Reply:
Good point, this was a little inaccurate so I've updated it.  Good to get it right here as we will likely copy paste this to a few models.